common/common.scss

```diff
@@ -1,6 +1,6 @@
 .d-wrap[data-wrap="template"] {
   padding: 0.5em;
-  border: 1px dashed $primary-medium;
+  border: 5px solid var(--primary-low, $primary-low);
```

Comment:
Hm, it's `var(--primary-low, $primary-low)` but just `var(--primary-low)` below? Should probably stick to one version?

Reply:
yes I should add fallback here too :+1: thanks